tests/infrared/17.1/README.md

```diff
@@ -0,0 +1,16 @@
+# Deployments
+
+## Basic deployment
+
+```bash
+CA_CERT_FILE_CONTENT="$(oc get secret/default-interconnect-selfsigned -o jsonpath='{.data.ca\.crt}' | base64 -d)" \
+OCP_ROUTE_IP="10.0.100.50" \
+AMQP_HOST="default-interconnect-5671-service-telemetry.apps.stf15.localhost" \
```

Comment:
No fancy `oc get route` commands for these ones? ;) It's proably clearer like this.

Reply:
Haha nope! Although that's a nice change request :)